modules/product/src/main/java/com/opengamma/strata/product/swap/type/StandardFixedInflationSwapConventions.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.swap.type;
+
+
+import static com.opengamma.strata.basics.currency.Currency.USD;
+import static com.opengamma.strata.basics.currency.Currency.GBP;
+import static com.opengamma.strata.basics.currency.Currency.EUR;
+import static com.opengamma.strata.basics.currency.Currency.CHF;
+import static com.opengamma.strata.basics.currency.Currency.JPY;
+import static com.opengamma.strata.basics.date.BusinessDayConventions.MODIFIED_FOLLOWING;
+import static com.opengamma.strata.basics.date.DayCounts.THIRTY_U_360;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.CHZU;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.EUTA;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.GBLO;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.JPTO;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.USNY;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.FRPA;
+import static com.opengamma.strata.basics.schedule.Frequency.P6M;
+
+import com.opengamma.strata.basics.date.BusinessDayAdjustment;
+import com.opengamma.strata.basics.date.DaysAdjustment;
+import com.opengamma.strata.basics.index.PriceIndices;
+
+/**
+ * Fixed-Inflation swap conventions.
+ */
+final class StandardFixedInflationSwapConventions {
+
+  /**
+   * GBP vanilla fixed vs UK HCIP swap.
+   * The fixed leg pays every 6 months with day count '30U/360'.
+   */
+  public static final FixedInflationSwapConvention GBP_FIXED_6M_GB_HCIP =
+      ImmutableFixedInflationSwapConvention.of(
+          "GBP-FIXED-6M-GB-HCIP",
+          FixedRateSwapLegConvention.of(GBP, THIRTY_U_360, P6M, BusinessDayAdjustment.of(MODIFIED_FOLLOWING, GBLO)),
```

Comment:
The fixed leg of zero-coupon inflation swaps as only one final payment with annual compounding. See SwapDummyData#INFLATION_FIXED_SWAP_LEG_PAY_GBP. The paymentSchedule.paymentFrequency should be "Frequency.TERM".

Same for all other conventions.